scripts/fastapi/routes/admin.py

```
@@ -153,3 +153,80 @@ async def get_log_content(
+@router.get("/cache/dump")
+async def trigger_cache_dump(
+    api_key: str = Depends(verify_api_key_query)
+):
+    """Manually trigger cache dump to DynamoDB
+
+    Access via: /admin/cache/dump?api_key=YOUR_API_KEY
+    """
+    try:
+        from cache_dumper import dump_cache_to_db
+        result = await dump_cache_to_db()
+        return {"success": True, "data": result}
```

Ruff (0.14.5)

Consider moving this statement to an `else` block

(TRY300)